### PR TITLE
Remove hover for active tab

### PR DESCRIPTION
**Pull request by MewenLeHo**
Closes [#1332](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1332)

### CODE REVIEW

**Review by isabellechanclou — APPROVED** [review]
Both fixes ok for me.

**Comment by julien-deramond** [comment]
It is said in the issue not to take into account focus modifications here. But still there is a regression when an active item is focused:
Capture d’écran 2022-07-29 à 13 50 12

It should remain orange.

File: scss/_nav.scss [hunk 1]
```diff
@@ -127,7 +127,7 @@
     // stylelint-disable-next-line function-disallowed-list
     border-width: 0 0 calc(var(--#{$prefix}nav-tabs-border-width) * 4);
 
-    &:hover,
+    &:not(.active):hover,
```
Comment (julien-deramond) [inline scss/_nav.scss:130]: IDK if this rule is useful.

Comment (MewenLeHo) [reply scss/_nav.scss]: It was indeed useful for the previous version of the issue.
I had to revamp it a bit for the updated version.

File: scss/_nav.scss [hunk 2]
```diff
@@ -135,6 +135,10 @@
       outline-color: currentcolor;
     }
 
+    &.active:hover {
+      color: var(--#{$prefix}nav-tabs-link-active-color);
+    }
+
```
Comment (isabellechanclou) [inline scss/_nav.scss:141]: OK with this.

 ⚠️ There is a remaining visual behavior to be fixed in PR #1437: on mouse down (active state) on a selected item, the text color stays black as expected, but when the click is released and the mouse is moved away, 
the text color turns orange when it shouldn't. It has to do with the focus management.

**Comment by julien-deramond** [comment]
Regression: Active + focused tab item changes color (+ border color) when hovered. 

Peek 2022-08-24 14-13

**Comment by louismaximepiton** [comment]
It should maybe be merged once the #1437 is merged ?

File: scss/_nav.scss [hunk 3]
```diff
@@ -135,6 +135,10 @@
       outline-color: currentcolor;
     }
 
+    &.active:hover {
+      color: var(--#{$prefix}nav-tabs-link-active-color);
```
Comment (louismaximepiton) [inline scss/_nav.scss:139]: Why not writing this line inside the `.active` below ?

**Review by julien-deramond — APPROVED** [review]
LGTM!

**Pull request merged**